### PR TITLE
Implement browsing files more user friendly

### PR DESCRIPTION
**Pull request by AidasVaiksnoras**

### CODE REVIEW

File: Foosball2text/NavigationForm.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        }
+
+        private void button1_Click(object sender, EventArgs e)
+        {
+            DialogResult result = openFileDialog1.ShowDialog();
+        }
+
+        private void openFileDialog1_FileOk(object sender, CancelEventArgs e)
+        {
+            this.Hide();
```
Comment (WarHatch) [inline Foosball2text/NavigationForm.cs]: I will want to test if program closes correctly. Sometimes there's a coding error when a user is able to close one form but the hidden form remains active

Comment (AidasVaiksnoras) [reply Foosball2text/NavigationForm.cs]: I believe there is this issue, I'll look into this on next commit, I just want all contributors to have basic UI

Comment (WarHatch) [reply Foosball2text/NavigationForm.cs]: You can either close this window or you need to have a reference to the hidden window in the next form you open

Comment (AidasVaiksnoras) [reply Foosball2text/NavigationForm.cs]: Yeah I see your point

**Pull request merged**